docs/source/conf.py

```diff
@@ -257,10 +257,21 @@
     load_profile(config.default_profile_name)
     get_manager().get_backend()
 else:
-    # Back-end settings for readthedocs online documentation.
     from aiida.manage import configuration
+    from aiida.manage.configuration import load_profile, reset_config
+    from aiida.manage.manager import get_manager
+
+    # Set the global variable to trigger shortcut behavior in `aiida.manager.configuration.load_config`
     configuration.IN_RT_DOC_MODE = True
-    configuration.BACKEND = 'django'
+
+    # First need to reset the config, because an empty one will have been loaded when `aiida` module got imported
+    reset_config()
+
+    # Load the profile: this will first load the config, which will be the dummy one for RTD purposes
+    load_profile()
+
+    # Finally load the database backend but without checking the schema because there is no actual database
+    get_manager()._load_backend(schema_check=False)
```

Comment:
@sphuber One question: I could imagine that loading a dummy profile could be useful not just on read the docs but in other contexts as well (e.g. for a quick test that doesn't need the DB... perhaps others).

So I'm wondering: instead of this slightly hackish way of getting there, how about we either
 * add a function `load_dummy_profile()` or
 * add an optional argument `dummy_profile` (or similar) to`load_profile` 

that does the job?
Then people who need this (even if just to enable building their docs on RTD) don't need to know about AiiDA internals.

Reply:
I guess that makes sense. I didn't do it now because as soon as we make this an "official" public function we should now what potential problems and dangers could be. But I am very willing to discuss this and see if we can implement a useful solution